ethcore/src/miner/miner.rs

```diff
@@ -277,13 +276,19 @@ impl MinerService for Miner {
 				// give the sender max balance
 				state.sub_balance(&sender, &balance);
```

Comment:
right. in that case it should be filed as a new PR (my fault for including a half a fix, but that ship has sailed now).